persistence/sql/migrations/20200527215731_client.cockroach.down.sql

```diff
@@ -1 +1 @@
- ALTER TABLE hydra_client DROP COLUMN token_endpoint_auth_signing_alg;
+ALTER TABLE hydra_client DROP COLUMN token_endpoint_auth_signing_alg;
```

Comment:
This is correct - basically you can rename this file to `20200527215731_client.down.sql` and override the existing file. By having `ALTER` and `UPDATE` in two files this should work for all migrations :)